### ISSUE TITLE
Command center should be centered when menu is visible

### ISSUE THREAD

**Issue by miguelsolorio** [microsoft/vscode#150984]
Testing #150396

![image](https://user-images.githubusercontent.com/35271042/171448191-ec1551bf-b06b-49ae-8353-391b75a45c73.png)

When using the command center on the web or windows, after a certain breakpoint the command center becomes off-centered and doesn't look great. I also notice that as you make the window smaller, the menu overflows into the ellipsis:

<img width="629" alt="CleanShot 2022-06-01 at 08 32 46@2x" src="https://user-images.githubusercontent.com/35271042/171442890-9b61e1dc-edc7-47bd-b99d-10e49e6af149.png">

## Suggestion
My suggestion would be to keep the command center centered (heh) and make the menu go into an ellipsis:

<img width="1090" alt="CleanShot 2022-06-01 at 08 35 37@2x" src="https://user-images.githubusercontent.com/35271042/171443472-ba79de34-95f6-4f95-8146-634d72d69d27.png">

**Comment by safasofuoglu** [comment]
@misolori would you consider the logo taking the role of the hamburger button when menu is hidden? I believe this suggestion becomes more relevant with the title bar height change (https://github.com/microsoft/vscode/pull/150272), since it makes room for a bigger logo.

**Comment by misolori** [comment]
@safasofuoglu I don't think so, in the past we experimented with making the logo that menu and it wasn't a clear enough distinction, so we opted to introduce the compact menu option that has a menu button.

**Comment by jrieken** [comment]
Yeah, the command center very much copies what we were doing for the static title already.. I do like the idea of the burger menu and also the idea of the ellipsis. Question is if this is a command center special case or if it should also be done for the static title?

**Comment by sbatten** [comment]
With the static title, it made most sense to me to prioritize space for the menu since it required interaction and we were migrating from an experience where the menu was fully visible always. With both the command center and menu, I would go straight to hamburger menu as it feels like the right compromise and is much less error-prone.

After having years on the custom menu, I could be convinced that the static title could follow suit, but I suspect there would be push back.

**Comment by safasofuoglu** [comment]
I'd like to nominate slack title bar for spaciousness and collapsed hamburger menu.

![image](https://user-images.githubusercontent.com/1001335/174439930-4c11c8e9-2d15-414c-a0fb-70bbb7593f2f.png)

**Comment by byehack** [comment]
> I'd like to nominate slack title bar for spaciousness and collapsed hamburger menu.
> 
> ![image](https://user-images.githubusercontent.com/1001335/174439930-4c11c8e9-2d15-414c-a0fb-70bbb7593f2f.png)


By this way isn't it better to remove vscode icon on left?

![image](https://user-images.githubusercontent.com/26390254/176895361-d174f539-d420-46c2-a6a8-44e84d815559.png)

**Comment by manojbaishya** [comment]
Can we please have the "Full Menu Bar + Command Center" on 2K screens or above in maximized mode and collapse the menu bar to a hamburger only after a breakpoint?

I remember the above being the case during the experimental phase.

**Comment by sbatten** [comment]
@manojbaishya this request was made in a separate issue and is the direction we take in the latest insiders builds

**Comment by sbatten** [comment]
This has been addressed with a separate change motivated by performance.
